### ISSUE TITLE
Pass and filter vocabulary items via vocab servlet

### ISSUE THREAD

**Issue by twagoo** [clarin-eric/component-registry-rest#26]
Facilitating for https://github.com/clarin-eric/component-registry-front-end/issues/63, add a call to retrieve the results of 'find-concepts' filtered by concept scheme.

Concepts can be obtained via 
```
{api-base}/find-concepts?q=prefLabel:*&conceptScheme={schemeUri}[&format=json]
```
where `{schemeUri}` is obtained from the `rdf:about` in the [concept scheme listing](http://145.100.58.79/clavas/public/api/conceptscheme)

Example: [find-concepts?q=prefLabel:*&conceptScheme=http://openskos.meertens.knaw.nl/iso-639-3&format=json](http://145.100.58.79/clavas/public/api/find-concepts?q=prefLabel:*&conceptScheme=http://openskos.meertens.knaw.nl/iso-639-3&format=json)

Since the default result is rather rich in information, make sure only the required fields are returned and/or filtered out before passing it to the client. This should be possible using the query syntax (lucene).

[jolt](https://github.com/bazaarvoice/jolt) can potentially be used (as an alternative  or in addition to query time filtering) for pre-processing the response.

**Comment by twagoo** [comment]
Vocabularies larger than 5000 items (e.g. language codes) are not forwarded entirely. Revise retrieval strategy...

**Comment by twagoo** [comment]
Fixed as of 306381f
